Sources/Plasma/FeatureLib/pfPython/pyWaveSet.h

```diff
@@ -185,6 +185,13 @@ class pyWaveSet
 
     PyObject* GetEnvCenter() const; // returns pyPoint3
     float GetEnvRadius() const;
+
+    // ==============================================================================
+    // Buoy functions
+    // ==============================================================================
+
+    void AddBuoy(pyKey& soKey);
+    void RemoveBuoy(pyKey& soKey);
```

Comment:
I feel like these can be const references (and the methods themselves could also be const).